.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
-public
\ No newline at end of file
+public
+coverage
+__snapshots__
```

Comment:
do not ignore `__snapshots__` folder, they should work as indicator that something has changed. To update snapshots you can run jest with `--updateSnapshots` flag